### PR TITLE
feature(whl): add more DingEnvWrapper example

### PR DESCRIPTION
**Pull request by kxzxvbk**
## Check List

- [ ] merge the latest version source branch/repo, and resolve all the conflicts
- [ ] pass style check
- [ ] pass all the tests

### CODE REVIEW

File: dizoo/procgen/entry/coinrun_ppo_main.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+def main(cfg, seed=0, max_iterations=int(1e10)):
+    cfg = compile_config(
+        cfg, BaseEnvManager, PPOPolicy, BaseLearner, SampleSerialCollector, InteractionSerialEvaluator,
+        save_cfg=True
+    )
+    collector_env_num, evaluator_env_num = cfg.env.collector_env_num, cfg.env.evaluator_env_num
+    collector_env = BaseEnvManager(env_fn=[lambda: wrapped_procgen_env(coinrun_ppo_default_config.env) for _ in range(collector_env_num)],
+                                   cfg=cfg.env.manager)
+    evaluator_env = BaseEnvManager(env_fn=[lambda: wrapped_procgen_env(coinrun_ppo_default_config.env) for _ in range(evaluator_env_num)],
```
Comment (puyuan1996) [inline dizoo/procgen/entry/coinrun_ppo_main.py:80]: 使用`partial(wrapped_procgen_env, cfg=coinrun_ppo_default_config.env)`？

File: dizoo/procgen/entry/coinrun_ppo_main.py [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+from ding.model import VAC
+from ding.policy import PPOPolicy
+from ding.envs import DingEnvWrapper, FinalEvalRewardEnv, BaseEnvManager
+from ding.config import compile_config
+from ding.utils import set_pkg_seed
+
+from dizoo.procgen.envs import ProcgenEnv
+from tensorboardX import SummaryWriter
+import os
+from dizoo.procgen.config.coinrun_ppo_config import coinrun_ppo_default_config
```
Comment (puyuan1996) [inline dizoo/procgen/entry/coinrun_ppo_main.py:11]: 参考我们之前的[config规范](https://di-engine-docs.readthedocs.io/zh_CN/latest/20_spec/config_spec_zh.html#id4) `coinrun_ppo_default_config` procgen/config里面的default全部去掉？

File: dizoo/dmc2gym/entry/ppo_main.py [hunk 3]
```diff
@@ -0,0 +1,125 @@
+from ding.worker import BaseLearner, SampleSerialCollector, InteractionSerialEvaluator
```
Comment (puyuan1996) [inline dizoo/dmc2gym/entry/ppo_main.py:1]: 文件命名为`dmc2gym_onppo_main.py`

File: dizoo/dmc2gym/entry/ppo_main.py [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+
+def main(cfg, seed=0, max_iterations=int(1e10)):
+    cfg = compile_config(
+        cfg, BaseEnvManager, PPOPolicy, BaseLearner, SampleSerialCollector, InteractionSerialEvaluator,
+        save_cfg=True
+    )
+    collector_env_num, evaluator_env_num = cfg.env.collector_env_num, cfg.env.evaluator_env_num
+    collector_env = BaseEnvManager(env_fn=[lambda: wrapped_dmc2gym_env(cartpole_balance_ppo_config.env) for _ in range(collector_env_num)],
+                                   cfg=cfg.env.manager)
+    evaluator_env = BaseEnvManager(env_fn=[lambda: wrapped_dmc2gym_env(cartpole_balance_ppo_config.env) for _ in range(evaluator_env_num)],
```
Comment (puyuan1996) [inline dizoo/dmc2gym/entry/ppo_main.py:93]: 使用`partial(wrapped_dmc2gym_env, cfg=cartpole_balance_ppo_config.env)`？

File: dizoo/dmc2gym/entry/ppo_main.py [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+                                   cfg=cfg.env.manager)
+    # collector_env = BaseEnvManager(env_fn=[lambda: DMC2GymEnv(cfg.env) for _ in range(collector_env_num)],
```
Comment (puyuan1996) [inline dizoo/dmc2gym/entry/ppo_main.py:95]: 去掉？

File: dizoo/rocket/entry/new_hover.py [hunk 6]
```diff
@@ -0,0 +1,97 @@
+from turtle import Terminator
```
Comment (puyuan1996) [inline dizoo/rocket/entry/new_hover.py:1]: 命名为`rocket_hover_onppo_main_v2`?

File: dizoo/dmc2gym/config/dmc2gym_ppo_config.py [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+    env=dict(
+        type='dmc2gym',
+        import_names=['dizoo.dmc2gym.envs.dmc2gym_env'],
+    ),
+    env_manager=dict(type='base'),
+    policy=dict(type='ppo'),
+    replay_buffer=dict(type='naive', ),
+)
+cartpole_balance_create_config = EasyDict(cartpole_balance_create_config)
+create_config = cartpole_balance_create_config
```
Comment (PaParaZz1) [inline dizoo/dmc2gym/config/dmc2gym_ppo_config.py:62]: add comments for which entry file can be called to use this config

File: dizoo/dmc2gym/entry/dmc2gym_onppo_main.py [hunk 8]
```diff
@@ -0,0 +1,122 @@
...
+from dizoo.dmc2gym.config.dmc2gym_ppo_config import cartpole_balance_ppo_config
+from dizoo.dmc2gym.envs.dmc2gym_env import *
+
+
+class InfoWrapper(gym.Wrapper):
+    def __init__(self, env, cfg):
+        super().__init__(env)
+        cfg = EasyDict(cfg)
+        self._cfg = cfg
+        self._observation_space = dmc2gym_env_info[cfg.domain_name][cfg.task_name]["observation_space"](
```
Comment (PaParaZz1) [inline dizoo/dmc2gym/entry/dmc2gym_onppo_main.py:22]: reuse varaible `dmc2gym_env_info[cfg.domain_name][cfg.task_name]`

File: dizoo/dmc2gym/entry/dmc2gym_onppo_main.py [hunk 9]
```diff
@@ -0,0 +1,122 @@
...
+            from_pixels=self._cfg["from_pixels"],
+            height=self._cfg["height"],
+            width=self._cfg["width"],
+            channels_first=self._cfg["channels_first"]
+        )
+        self._action_space = dmc2gym_env_info[cfg.domain_name][cfg.task_name]["action_space"]
+        self._reward_space = dmc2gym_env_info[cfg.domain_name][cfg.task_name]["reward_space"](self._cfg["frame_skip"])
+
+    def seed(self, seed: int, dynamic_seed: bool = True) -> None:
```
Comment (PaParaZz1) [inline dizoo/dmc2gym/entry/dmc2gym_onppo_main.py:31]: why add this method, is it added in `DingEnvWrapper`?

File: dizoo/dmc2gym/entry/dmc2gym_onppo_main.py [hunk 10]
```diff
@@ -0,0 +1,122 @@
...
+from ding.model import VAC
+from ding.policy import PPOPolicy
+from ding.envs import DingEnvWrapper, FinalEvalRewardEnv, BaseEnvManager
+from ding.config import compile_config
+from ding.utils import set_pkg_seed
+from dizoo.dmc2gym.config.dmc2gym_ppo_config import cartpole_balance_ppo_config
+from dizoo.dmc2gym.envs.dmc2gym_env import *
+
+
+class InfoWrapper(gym.Wrapper):
```
Comment (PaParaZz1) [inline dizoo/dmc2gym/entry/dmc2gym_onppo_main.py:17]: why name this class `InfoWrapper`

File: dizoo/dmc2gym/entry/dmc2gym_onppo_main.py [hunk 11]
```diff
@@ -0,0 +1,122 @@
...
+        cfg={
+            'env_wrapper': [
+                lambda env: InfoWrapper(env, default_cfg),
+                lambda env: FinalEvalRewardEnv(env),
+            ]
+        }
+    )
+
+
+def main(cfg, seed=0, max_iterations=int(1e10)):
```
Comment (PaParaZz1) [inline dizoo/dmc2gym/entry/dmc2gym_onppo_main.py:86]: don't use `max_iterations` argument but set `max_env_step` like that in `serial_pipeline`

**Review by PaParaZz1 — CHANGES_REQUESTED** [review]

**Review by PaParaZz1 — APPROVED** [review]

**Pull request merged**